src/pages/javascript/standard-objects/string/string-prototype-trim/index.md

```diff
@@ -3,13 +3,18 @@ title: String.prototype.trim
 ---
 ## String.prototype.trim
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/standard-objects/string/string-prototype-trim/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+The `trim()` function removes any whitespace characters from both the beginning and the end of a given string. It does not modify the original string; it outputs a new one.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+Examples:
+```js
+"  Hello, campers. I have spaces on both ends!  ".trim();
+// output is "Hello, campers. I have spaces on both sides!"
```

Comment:
The output should be "on both ends!" not "on both sides!".

Reply:
Ah, of course! 😅 Will fix now.